flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/TwoPhaseCommitSinkFunction.java

```diff
@@ -230,11 +239,16 @@ public final void invoke(IN value) throws Exception {}
 
     @Override
     public final void invoke(IN value, Context context) throws Exception {
-        invoke(currentTransactionHolder.handle, value, context);
+        TXN currentTransaction = currentTransaction();
+        checkNotNull(
+                currentTransaction,
+                "two phase commit sink function with null transaction should not be invoked! ");
```

Comment:
minor: Capitalise the first word? Remove the last extra space after the exclamation point?